go.mod

```diff
@@ -71,10 +71,10 @@ require (
 	golang.org/x/time v0.0.0-20190921001708-c4c64cad1fd0
 	google.golang.org/appengine v1.6.1 // indirect
 	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
-	k8s.io/api v0.18.0
+	k8s.io/api v0.18.6
 	k8s.io/apiextensions-apiserver v0.18.0
```

Comment:
I think that we should bring all k8s dependencies to the same level, `k8s.io/api/apiextensions-apiserver` as well.
Please check for others.